docs/src/main/asciidoc/_developer-guide/JPA_Reference_Guide.adoc

```diff
@@ -20,24 +20,20 @@ deploy JPA applications.
 
 In the remainder of this documentation, "entity manager" refers to an
 instance of the _javax.persistence.EntityManager_ class.
-http://download.oracle.com/javaee/7/api/javax/persistence/package-summary.html[Javadoc
-for the JPA interfaces]and https://jcp.org/en/jsr/detail?id=338[JPA 2.1
+https://javaee.github.io/javaee-spec/javadocs/javax/persistence/package-summary.html[Javadoc
```

Comment:
Should this link be used instead: https://javaee.github.io/javaee-spec/javadocs/overview-summary.html ? That page shows the title, "Java(TM) EE 8 Specification APIs"

Reply:
Yes, good catch!  updating...